### DISCUSSION TITLE
Duplicate settings?

### DISCUSSION THREAD

**Discussion by chinghwa** [change-metrics/monocle#1118] [Q&A] [answered]
In the Environment section, https://github.com/change-metrics/monocle?tab=readme-ov-file#environment-variables
it mentions `COMPOSE_MONOCLE_PUBLIC_URL`

However in the OIDC provider section, https://github.com/change-metrics/monocle?tab=readme-ov-file#oidc-provider
it mentions `MONOCLE_PUBLIC_URL`

If I want to enable OIDC auth, do I need both? 😕

**Comment by morucci** [comment, marked as answer]
Hi, if you deploy via the compose you just set COMPOSE_MONOCLE_PUBLIC_URL in the env file. Please check the docker-compose.yml file.

**Reply by chinghwa** [reply]
ah yes, I see it now:
```
MONOCLE_PUBLIC_URL: "${COMPOSE_MONOCLE_PUBLIC_URL:-http://localhost:8080}"
```
